### PR TITLE
Added option to return 'No such file or directory' error

### LINKED ISSUE #67137: Crash when use OS.move_to_trash repeatedly
### Godot version

v3.5.1.stable.official [6fed1ffa3]

### System information

w10 64

### Issue description

Little to say, the program closes when the red button is pressed (OS.move_to_trash())
The problem happens on windows.
Ignore the attached project folder name "testAndroid".

Captura

Captura2

### Steps to reproduce

Execute and press red button.

### Minimal reproduction project

[testAndroid.zip](https://github.com/godotengine/godot/files/9741806/testAndroid.zip)

### PR DESCRIPTION
**Pull request by strangezakary**
Fixes #67137

First PR please let me know if anything seems wrong :)

### CODE REVIEW

File: platform/windows/os_windows.cpp [hunk 1]
```diff
@@ -1115,6 +1115,11 @@ Error OS_Windows::move_to_trash(const String &p_path) {
 	delete[] from;
 
 	if (ret) {
+		if(ret == 2) {
```
Comment (Calinou) [inline platform/windows/os_windows.cpp:1118]: Fix formatting:

```suggestion
		if (ret == 2) {
```

Comment (StrangeZak) [reply platform/windows/os_windows.cpp]: Fixed

File: platform/windows/os_windows.cpp [hunk 2]
```diff
@@ -1115,6 +1115,11 @@ Error OS_Windows::move_to_trash(const String &p_path) {
 	delete[] from;
 
 	if (ret) {
+		if (ret == 2) {
```
Comment (bruvzg) [inline platform/windows/os_windows.cpp:1118]:
```suggestion
		if (ret == ERROR_FILE_NOT_FOUND) {
```

**Review by bruvzg — COMMENTED** [review]
> Fixes https://github.com/godotengine/godot/issues/67137

How exactly it's supposed to fix a crash? It's going to print a more readable error, but otherwise the behavior is the same.

**Comment by YuriSizov** [comment]
Closing as this doesn't solve the issue, as per comment above.

**Pull request closed without merging**